src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -1010,6 +1007,7 @@ public static void EmitObject(string objectFilePath, IEnumerable<DependencyNode>
                     objectWriter.SetSection(section);
                     objectWriter.EmitAlignment(nodeContents.Alignment);
 
+                    objectWriter._byteInterruptionOffsets = new bool[nodeContents.Data.Length + 1];
```

Comment:
Can we reallocate the array only if it is not big enough, and just zero-fill it otherwise?